### DISCUSSION TITLE
[TID:N/A ] is emptiy

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11846] [Q&A]
The tid in the log is empty，why?
 
<img width="1304" alt="企业微信截图_2f199c64-c6dd-476b-acd4-5e1424b52a95" src="https://github.com/apache/skywalking/assets/31468018/01abf3e6-8923-4f75-8f7f-fb56c657acff">

**Comment by CodePrometheus** [comment]
I think the possible reason is that the tracer is inactive, or the method you used, the amqp plugin of Java agent does not intercept

**Comment by Superskyyy** [comment]
Not all logs have a TID associated with it, it could be emitted by something else that is not traced.
